src/app/content/components/Topbar/index.tsx

```diff
@@ -85,28 +91,52 @@ class Topbar extends React.Component<Props, State> {
       this.props.openSearchResults();
     };
 
-    const onChange = (e: React.FormEvent<HTMLInputElement>) => {
+    const onSearchChange = (e: React.FormEvent<HTMLInputElement>) => {
       this.setState({ query: (e.currentTarget as any).value, formSubmitted: false });
     };
 
+    const onChangeTextSize = (e: React.FormEvent<HTMLInputElement>) => {
+      const target = (e as any).currentTarget;
+      const value = parseInt(target.value, 10);
+
+      this.props.setTextSize(value);
+    };
+
+    const onDecreaseTextSize = (e: React.FormEvent<HTMLInputElement>) => {
+      e.preventDefault();
+      const newValue = (this.props.textSize || textResizerDefaultValue) - 1;
+      if (newValue < textResizerMinValue) { return; }
+      this.props.setTextSize(newValue);
+    };
+
+    const onIncreaseTextSize = (e: React.FormEvent<HTMLInputElement>) => {
+      e.preventDefault();
+      const newValue = (this.props.textSize || textResizerDefaultValue) + 1;
+      if (newValue > textResizerMaxValue) { return; }
+      this.props.setTextSize(newValue);
+    };
+
     const showBackToSearchResults = !this.props.searchSidebarOpen && this.props.hasSearchResults;
 
     const newButtonEnabled = !!this.props.searchButtonColor;
 
     return <Styled.TopBarWrapper data-testid='topbar'>
+      {/* mobileMedium is smaller (800px) than mobile (1200px) */}
+      {/* Only hides on .mobile breakpoint, shows on desktop and .mobileMedium */}
```

Comment:
what hides on mobile? does this comment conflict with the one on line 175?

Reply:
Styled.SearchPrintWrapper below, but these were more to keep it straight for myself while working on it, might just remove them because they feel prone to being out of date soon